routes/cp.php

```diff
@@ -27,7 +27,16 @@
     Route::get('select-site/{handle}', 'SelectSiteController@select');
 
     Route::group(['namespace' => 'Structures'], function () {
-        Route::resource('navigation', 'NavigationController');
+        Route::resource('navs', 'NavigationController', ['names' => [
+            'index' => 'navigation.index',
+            'create' => 'navigation.create',
+            'store' => 'navigation.store',
+            'show' => 'navigation.show',
+            'edit' => 'navigation.edit',
+            'update' => 'navigation.update',
+            'destroy' => 'navigation.destroy',
+        ]]);
+        Route::redirect('navigation/{any?}', redirect(str_replace_first('navigation', 'navs', Request::fullUrl())));
```

Comment:
This doesn't seem to work.
When I hit `/cp/navigation`, I get sent to `/cp/HTTP/1.0%20302%20Found%0D%0ACache-Control:%20no-cache,%20private...`

This breaks route caching anyway. Maybe we just scrap this redirect.

Reply:
Weird, it works fine for me here 🤔

Reply:
Not sure why you are getting sent there. It might be that I had a `redirect()` call in there? Anyway, just pushed some changes...

A) Does the redirect work for you now?
B) Is route caching still broken? I can `route:cache` and `route:clear` without issues here now.

Reply:
That fixed the weirdness in the URL, yup.

The error when caching routes is gone now, but the redirect doesn't work.
You can't have a str_replace right in there.